crates/deno-subsystem/deno-model-builder/src/module_skeleton_generator.rs

```diff
@@ -595,10 +655,13 @@ mod tests {
 
         let expected_type1 = "export interface TestType1 {\n\tfield1: string\n\tfield2: number\n}";
         let expected_type2 = "export interface TestType2 {\n\tfield1: string\n\tfield2: number\n}";
+        let expected_edge_type =
+            "export interface EdgeType {\n\titems: Item[]\n\ttotalCount: number\n}";
         assert!(content.contains(expected_type1), "TestType1 not found");
         assert!(content.contains(expected_type2), "TestType2 not found");
+        assert!(content.contains(expected_edge_type), "EdgeType not found");
 
-        fs::remove_dir_all(generated_dir).unwrap();
+        // fs::remove_dir_all(generated_dir).unwrap();
```

Comment:
Looks like this was commented during development to see the result. Needs to be uncommented.

Reply:
wops!